playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -192,7 +192,7 @@
     when: origin_reconcile_bindings | bool or ent_reconcile_bindings | bool
     register: reconcile_bindings_result
     changed_when:
-    - reconcile_bindings_result.stdout.length > 0
+    - reconcile_bindings_result.stdout != ''
```

Comment:
ack, that's pretty cool; i didn't realize that we could do that. So it would be something like 
```
changed_when:
- not reconcile_bindings_results.stdout
```

However I think this makes the check non-obvious for people that don't realize you can do that. It would probably be better if we had a way to test like 
```
changed_when:
- not reconcile_bindings_result.stdout | empty
```